setup.sh

```diff
@@ -110,6 +110,17 @@ setup_networking() (
 	else
 		echo "$ERR tinkerbell network interface configuration failed"
 	fi
+
+	if [ -r .nat_interface ]; then
+		NAT_INTERFACE=$(cat .nat_interface)
+	fi
+	if [ -n "$NAT_INTERFACE" ] && ip addr show "$NAT_INTERFACE" &>/dev/null; then
+		# TODO(nshalman) the terraform code would just run these commands as-is once
```

Comment:
ubuntu has some sort of iptables-persist package iirc. Yep `iptables-persistent`

Reply:
Should I try to get that wired up, or land as is with the TODO?

Reply:
todo is fine imo, hasn't been an issue so far I think.